bindings/python/src/OpenSpaceToolkitAstrodynamicsPy.cxx

```diff
@@ -23,9 +23,6 @@ PYBIND11_MODULE(OpenSpaceToolkitAstrodynamicsPy, m)
     // Add optional docstring for package OpenSpaceToolkitAstrodynamicsPy
     m.doc() = "Orbit, attitude, access for OpenSpaceToolkit";
 
-    // Add __path__ attribute to python package
```

Comment:
Does removing this fix something?

Is this the reason for the weird auto-complete in a jupyter env?

Reply:
It's required for sphinx auto-generation as by overloading the __path__ field it cannot appropriately introspect the package and recursively find subpackages.